pageserver/src/http/routes.rs

```diff
@@ -1029,6 +1029,16 @@ async fn active_timeline_of_active_tenant(
         .map_err(ApiError::NotFound)
 }
 
+async fn always_panic_handler(_: Request<Body>) -> Result<Response<Body>, ApiError> {
+    // catch_unwind ensures that tokio nor hyper are distracted by our panic, but panic hooks
+    // (at least sentry) will report the issue. if tested across fleet, this should be a good test
+    // case for panic deduplication as well per version.
```

Comment:
```suggestion
    // Deliberately cause a panic to exercise the panic hook registered via std::panic::set_hook().
    // For pageserver, the relevant panic hook is `tracing_panic_hook` , and the `sentry` crate's wrapper around it.
    // Use catch_unwind to ensure that tokio nor hyper are distracted by our panic.
```

Reply:
Is this just stylistic rewrite or is there some changes I cannot see? unsure if std's panic hook needs to be mentioned, it's the only panic hook in std..

Reply:
Hmm ok, now I can see.